example/example.cpp

```diff
@@ -135,17 +133,14 @@ class TestSource2
     {
         isaac_float value = ptr[nIndex.x + nIndex.y * VOLUME_X
                                 + nIndex.z * VOLUME_X * VOLUME_Y];
-        isaac_float_dim< 1 > result;
-        result.value
-            .x = value;
-        return result;
+        return value;
```

Comment:
Here I am not sure if you can return `value` only. Value is a scalar. How is the implementation if you cast `isaac_float` to `isaac_float_dim <3>`? Is the scalar only assigned to `.x` or to all `3` members.

Reply:
It is fixed now, as feature dim is set to 1 the type is isaac_float_dim <1> and there is no implicit conversion from isaac_float to isaac_float_dim <1>